scripts/cors/cors.json

```diff
@@ -0,0 +1,13 @@
+{
+  "CORSRules": [
+    {
+      "AllowedOrigins": [
```

Comment:
It's probably worth a comment here explaining that this is consumed by the Service. Also, do we somehow suppress this functionality in self-hosted contexts? Since those won't be running on these domains, we should probably just make sure the UI won't break when it can't load this info.

Reply:
ahhh yeah that is a very good point! I imagine we would want to show to self hosted if possible as they are pulumi users so the events will likely be relevant to them.

Reply:
I am going to go ahead and hide the events dashboard card from self hosted folks for now. This is something we had wanted to rollout quickly to see what the response is like first and then plan to iterate on it more in the future, so its fine for now if we do not show to them.